rpc/api/debug_js.go

```diff
@@ -25,37 +25,37 @@ web3._extend({
 			name: 'printBlock',
 			call: 'debug_printBlock',
 			params: 1,
-			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter]
+			inputFormatter: [null]
```

Comment:
The old implementation is able to handle this. The old debug api uses the `BlockNumArg` that has a custom unmarshal method that can parse a number from a string or float64.